contrib/install-dependencies.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+apt update
+apt install genisoimage mtools python3-distro-info -y
+
+# if you want this tool to setup pxe server for you
+# these packages will be useful when invoking --deploy-dir
+apt install apache2 tftpd-hpa -y
```

Comment:
This is a little too scary to me. While nobody should be running scripts off the internet without understanding what they do - and certainly not as root as this requires - some people will, and I wouldn't want to open up listening services on their system w/o express permission.

Reply:
Understood. Let me strip `./contrib` and focus on `--autoinstall-url`.